examples/crowdfunding/tests/mandos_test.rs

```diff
@@ -0,0 +1,20 @@
+
+extern crate crowdfunding;
+use crowdfunding::*;
+use elrond_wasm::*;
+use elrond_wasm_debug::*;
+
+fn contract_map() -> ContractMap<TxContext> {
+    let mut contract_map = ContractMap::new();
+    contract_map.register_contract(
+        "file:../output/crowdfunding.wasm",
+        Box::new(|context| Box::new(CrowdfundingImpl::new(context))));
+    contract_map
+}
+
+#[test]
```

Comment:
I like keeping them in separate tests, easier to figure out what failed, and also helps with debugging.
The is a tool in elrond-wasm/tools/test-gen that can auto-generate these implementations, if they are too many.

Reply:
#[test]
fn test_crowdfunding1() {
    parse_execute_mandos("mandos/crowdfunding-fund-too-late.scen.json", &contract_map());
}

#[test]
fn test_crowdfunding2() {
    parse_execute_mandos("mandos/crowdfunding-fund.scen.json", &contract_map());
}

#[test]
fn test_crowdfunding3() {
    parse_execute_mandos("mandos/crowdfunding-init.scen.json", &contract_map());
}